packages/fxa-settings/src/pages/AccountRecoveryResetPassword/en.ftl

```diff
@@ -0,0 +1,8 @@
+password-link-expired = Reset password link expired
```

Comment:
```suggestion
## Account recovery reset password page

#  Appears when a link to reset password has expired
password-link-expired-header = Reset password link expired
```

Reply:
Add group level comment and individual level comment to provide context for localizers.

Suggest tweaking ID to also provide context on how this appears. (More important for something like "Create new password" below which could be confused for a button/link.)